apps/meteor/ee/tests/unit/apps/livechat-enterprise/server/lib/verifyContactChannel.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { expect } from 'chai';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+
+const modelsMock = {
+	LivechatContacts: {
+		updateContactChannel: sinon.stub(),
+	},
+	LivechatRooms: {
+		update: sinon.stub(),
+	},
+	LivechatInquiry: {
+		findOneReadyByContactId: sinon.stub(),
+		saveQueueInquiry: sinon.stub(),
+	},
+};
+
+const mergeContactsStub = sinon.stub();
+
+const { runVerifyContactChannel } = proxyquire.noCallThru().load('../../../../../../server/patches/verifyContactChannel', {
+	'../../../app/livechat/server/lib/Contacts': { mergeContacts: mergeContactsStub, verifyContactChannel: { patch: sinon.stub() } },
+	'../../../app/livechat/server/lib/QueueManager': { saveQueueInquiry: sinon.stub() },
+	'@rocket.chat/models': modelsMock,
+});
+
+describe('verifyContactChannel', () => {
+	beforeEach(() => {
+		modelsMock.LivechatContacts.updateContactChannel.reset();
+		modelsMock.LivechatRooms.update.reset();
+		modelsMock.LivechatInquiry.findOneReadyByContactId.reset();
+	});
+
+	afterEach(() => {
+		sinon.restore();
+	});
+
+	it('should be able to verify a contact channel', async () => {
```

Comment:
could we add a couple of negative/failing cases? 😬

Reply:
There's only one. I've just added it